### PR TITLE
added grpc support

### PR DESCRIPTION
**Pull request by tharun208**
gRPC support docs

### CODE REVIEW

File: docs/docs/0.7.0/policies/supported-protocols.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+:::
+::::
+
+## HTTP/2 support
+
+Kuma by default upgrades connection between Dataplanes to HTTP/2. If you want to enable HTTP/2 on connections between a dataplane and an application, use `protocol: http2` tag.
+
+## gRPC support
+
+If you want to enable gRPC on connections between a dataplane and an application, use `protocol: grpc` tag.
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.7.0/policies/supported-protocols.md:66]: `protocol:` -> `kuma.io/protocol:`

File: docs/docs/0.6.0/policies/traffic-log.md [hunk 2]
```diff
@@ -177,7 +177,7 @@ All access log _command operators_ are valid to use with both `TCP` and `HTTP` t
 
 If a _command operator_ is specific to `HTTP` traffic, such as `%REQ(X?Y):Z%` or `%RESP(X?Y):Z%`, it will be replaced by a symbol "`-`" in case of `TCP` traffic.
 
-Internally, `Kuma` [determines traffic protocol](../http-support-in-kuma) based on the value of `protocol` tag on the `inbound` interface of a `destination` `Dataplane`.
+Internally, `Kuma` [determines traffic protocol](../supported-protocols) based on the value of `protocol` tag on the `inbound` interface of a `destination` `Dataplane`.
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.6.0/policies/traffic-log.md:180]: you should not change old docs

File: docs/docs/0.7.0/policies/traffic-log.md [hunk 3]
```diff
@@ -294,7 +294,7 @@ All access log _command operators_ are valid to use with both `TCP` and `HTTP` t
 
 If a _command operator_ is specific to `HTTP` traffic, such as `%REQ(X?Y):Z%` or `%RESP(X?Y):Z%`, it will be replaced by a symbol "`-`" in case of `TCP` traffic.
 
-Internally, `Kuma` [determines traffic protocol](../http-support-in-kuma) based on the value of `protocol` tag on the `inbound` interface of a `destination` `Dataplane`.
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.7.0/policies/traffic-log.md:297]: is `http-support-in-kuma` file removed?

File: docs/.vuepress/site-config/sidebar-nav.js [hunk 4]
```diff
@@ -1109,7 +1109,7 @@ module.exports = {
         "policies/proxy-template",
         "policies/general-notes-about-kuma-policies",
         "policies/how-kuma-chooses-the-right-policy-to-apply",
-        "policies/http-support-in-kuma",
+        "policies/supported-protocols",
```
Comment (jakubdyszkiewicz) [inline docs/.vuepress/site-config/sidebar-nav.js:1112]: only the newest version should be changed

File: docs/docs/0.7.0/policies/traffic-log.md (hunk 3 above)
Comment (tharun208) [reply docs/docs/0.7.0/policies/traffic-log.md:297]: renamed `http-support-in-kuma` to `supported-protocols`

File: docs/.vuepress/site-config/sidebar-nav.js (hunk 4 above)
Comment (tharun208) [reply docs/.vuepress/site-config/sidebar-nav.js:1112]: okay

File: docs/docs/0.6.0/policies/traffic-log.md (hunk 2 above)
Comment (tharun208) [reply docs/docs/0.6.0/policies/traffic-log.md:180]: okay

**Comment by nickolaev** [comment]
@tharun208 would you please rebase this to the latest docs and apply the suggested changes.

**Comment by subnetmarco** [comment]
@nickolaev @tharun208 this has already been updated: https://kuma.io/docs/0.7.1/policies/protocol-support-in-kuma/#http-2-support

**Pull request closed without merging**